Automattic/jetpack
# Add the Instagram widget from wpcomsh

Fixes #1319 

This PR is to start a discussion around brining this widget to Jetpack from wpcomsh. Right now this code is duplicated in Simple and Atomic sites. I believe we should move the code to Jetpack in order to have a single source of truth for this code, reduce our maintenance burden and unify our platforms.

The widget worked without any changes to the code, but there will be some changes need to fix linting etc. I'm happy to make these changes but I'd like to get a sense of whether this could be merged before doing so.

#### Testing instructions:

* Get a Jetpack site on master running wpcomsh
* Add an Instagram widget
* Checkout this branch
* Disable wpcomsh
* Check that this widget still loads

#### Proposed changelog entry for your changes:

* Add an Instagram widget
